benchmarks/htap/lib/helpers.py

```diff
@@ -284,24 +298,16 @@ def random_phone_number(self, key):
 
 
 class TimestampGenerator:
-    def __init__(self, start_date, scale_factor, random):
+    def __init__(self, start_date, random, workload_scalar = 1.0):
         self.random = random
         self.current = start_date
-        self.increment = self.timestamp_distance(scale_factor)
-
-    @staticmethod
-    def timestamp_distance(scale_factor):
```

Comment:
Here I would like to have a comment (not in the code, in the PR) why we needed this change.